velox/expression/Expr.cpp

```diff
@@ -40,6 +41,20 @@ DEFINE_bool(
     "Whether to overwrite queryCtx and force the "
     "use of simplified expression evaluation path.");
 
+DEFINE_bool(
+    experimental_velox_save_input_on_fatal_signal,
+    false,
+    "This is an experimental flag that, when set to true, "
+    "serializes the input vector data and all the SQL expressions "
+    "in the ExprSet that is currently executing, whenever a fatal "
+    "signal is encountered. Enabling this flag makes the signal "
+    "handler async signal unsafe, so it should only be used for "
+    "debugging purposes. The vector and SQLs are serialized to files "
+    "in directories specified by either "
+    "'velox_save_input_on_expression_any_failure_path' or "
+    "'velox_save_input_on_expression_system_failure_path', "
+    "so either of these should be specified to enable this.");
```

Comment:
This final 'so either of these..' is redundant since you already mentioned in directories specified by either above.